LittleHelpBook/Server/Services/AirTableService.cs

```diff
@@ -105,6 +131,24 @@ public async Task<IEnumerable<Place>> GetPlacesPopulatedAsync()
 
                     place.Subcategories = null;
                 }
+                if (place.CatSubcats != null)
+                {
+                    foreach (var id in place.CatSubcats)
+                    {
+                        place.CatSubcatList.Add(await GetCatSubcatAsync(id));
```

Comment:
Places allready have categories and categories have subcategories, so this is redundant.  We can leave it for now, but it blots the data.  I'll make a seperate issues to remote it.